packages/grid/x-data-grid/src/joy/icons.tsx

```diff
@@ -0,0 +1,328 @@
+import * as React from 'react';
```

Comment:
Should we include a license here? https://lucide.dev/license

Reply:
Added at the top of the file.